Sami/Parser/ClassVisitor/PropertyClassVisitor.php

```diff
@@ -100,9 +100,7 @@ protected function parseProperty($tag)
         if (strlen($type)) {
             $property['hint'] = $type;
         }
-        if (strlen($description)) {
-            $property['description'] = $description;
-        }
+        $property['description'] = $description;
```

Comment:
I've now always assigning a description key, because it's always used in caller of this method.